submissions/VikTorNik/html-css-popup/style.scss

```diff
@@ -0,0 +1,179 @@
+@import "null_style.scss";
+
+body {
+    height: 100%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+}
+
+.wrapper {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.header {
+    background-color: rgb(68, 202, 206);
+}
+
+.nav {
+
+    .nav__menu {
+
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        height: 50px;
+
+        li {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)